build.rs

```diff
@@ -3,12 +3,9 @@ use std::env;
 include!("src/dist/triple.rs");
 
 fn from_build() -> Result<PartialTargetTriple, String> {
-    let triple = if let Ok(triple) = env::var("RUSTUP_OVERRIDE_BUILD_TRIPLE") {
-        triple
-    } else {
-        env::var("TARGET").unwrap()
-    };
     PartialTargetTriple::new(&triple).ok_or(triple)
+    let triple =
```

Comment:
This change looks wrong, defining `triple` after using it.

Reply:
@djc I don't understand how this commit has been created. Maybe `git absorb` did something dumb. The resulting function in the end works perfect though.